core/modules/file/views/views_handler_field_file_icon.inc

```diff
@@ -0,0 +1,137 @@
+<?php
+/**
+ * @file
+ * Definition of views_handler_field_file_icon.
+ */
+
+/**
+ * @ingroup views_field_handlers
+ */
+class views_handler_field_file_icon extends views_handler_field {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function option_definition() {
+    $options = parent::option_definition();
+    $options['icon_size'] = array('default' => 32);
+    $options['display_type'] = array('default' => 'svg');
+    return $options;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function options_form(&$form, &$form_state) {
+    parent::options_form($form, $form_state);
+    $form['icon_size'] = array(
+      '#type' => 'number',
+      '#title' => t('Icon size'),
+      '#min' => 8,
+      '#max' => 300,
+      '#default_value' => $this->options['icon_size'],
+      '#field_suffix' => 'px',
+    );
+    $form['display_type'] = array(
+      '#type' => 'radios',
+      '#title' => t('Display type'),
+      '#options' => array(
+        'svg' => t('As svg tag'),
+        'img' => t('As img tag'),
```

Comment:
```suggestion
        'svg' => t('As <code>&lt;svg&gt;<code> tag'),
        'img' => t('As <code>&lt;img&gt;<code> tag'),
```

Reply:
From a translator's perspective the resulting string's a nightmare. :stuck_out_tongue_winking_eye: 
Maybe I find a better solution. Currently the options look too similar, anyway.

Reply:
I like what you did in [`30f1aae` (#4778)](https://github.com/backdrop/backdrop/pull/4778/commits/30f1aae44ef477d6193c1fdff1b81fbf3ef2387a) much better 👍🏼